### DISCUSSION TITLE
Brand new project with Nuxt 3 or still better with Nuxt 2 until stable release ?

### DISCUSSION THREAD

**Discussion by gstucki** [nuxt/nuxt#18359] [Questions] [answered]
Hello,

We were waiting for the Nuxt 3 Public beta to come out, to start working on our brand new project. But as it is carefully mentionned in the **Introduction**, "Nuxt 3 is currently in beta, keep in mind that it is not yet production ready."

So is it better to start with Nuxt 2 then migrate to Nuxt 3 (with or without Nuxt Bridge) once stable release ? Or we can directly working on Nuxt 3 ? Any date on the release candidate or stable release ?

Thanks in advance for your response.

Gael

**Comment by shadowlion** [comment, marked as answer]
I would stick to using Nuxt 2 until it's a little mature - Nuxt 3 beta is a good opportunity for plugin and module devs outside of the core team to actually work on upgrading their own code such that when v3 get rid of a lot of bugs, you (as a power user) can use it with ease.

However, [that doesn't mean you can't start using the features yourself](https://v3.nuxtjs.org/getting-started/bridge) in v2.
